src/SOS/Strike/clrma.cpp

```diff
@@ -40,10 +40,7 @@ HRESULT CLRMACreateInstance(ICLRManagedAnalysis** ppCLRMA)
             return hr;
         }
     }
-    else
-    {
-        g_managedAnalysis->AddRef();
-    }
+    g_managedAnalysis->AddRef();
```

Comment:
Is this worth adding a null check on the check in line 38?

Reply:
It should never be null (famous last words).